package/MDAnalysis/coordinates/XDR.py

```diff
@@ -93,15 +109,15 @@ def _load_offsets(self):
             self._read_offsets(store=True)
             return
 
-        with open(fname, 'rb') as f:
-            data = {k: v for k, v in six.iteritems(np.load(f))}
+        data = read_numpy_offsets(fname)
 
         ctime_ok = getctime(self.filename) == data['ctime']
```

Comment:
I hoped we could ditch the `ctime` check, now that we have the extra protection layer. But I agree there wasn't huge consensus (maybe @dotsdl would reconsider?).